frontend/app/routes/$lang/_public/status/child.tsx

```diff
@@ -294,6 +296,9 @@ export default function StatusCheckerChild() {
           <ButtonLink id="cancel-button" variant="primary" type="button" routeId="$lang/_public/status/index" params={params} className="mt-12" endIcon={faChevronRight}>
             {t('status:child.check-another')}
           </ButtonLink>
+          <InlineLink routeId="$lang/_public/status/index" params={params} className="mt-6 block font-lato font-semibold">
```

Comment:
I feel this link should have the same font style as the rest of the links in the application.

```suggestion
          <InlineLink routeId="$lang/_public/status/index" params={params} className="mt-6 block">
```